app/src/main/java/space/taran/arknavigator/mvp/model/repo/stats/category/TagUsedTSStorage.kt

```diff
@@ -0,0 +1,55 @@
+package space.taran.arknavigator.mvp.model.repo.stats.category
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import space.taran.arknavigator.mvp.model.repo.stats.StatsCategory
+import space.taran.arknavigator.mvp.model.repo.stats.StatsEvent
+import space.taran.arknavigator.utils.Tag
+import timber.log.Timber
+import java.nio.file.Path
+import kotlin.io.path.Path
+import kotlin.io.path.inputStream
+import kotlin.io.path.notExists
+import kotlin.io.path.writeText
+
+class TagUsedTSStorage(
+    root: Path,
+    scope: CoroutineScope
+) : StatsCategoryStorage<Map<Tag, Long>>(root, scope) {
+    override val fileName = Path("tag-used-ts")
```

Comment:
Actually, "used" is ambiguous word here.. We should use either "queried" or "labeled" here. Word "used" would mean disjunction of those two.

So, metric "tag-queried-ts" should be updated when we tap or type a tag in Resource Grid (modifying current query). And metric "tag-labeled-ts" should be updated when we tap or type a tag in Gallery mode (editing tags for some resource).